src/main/java/com/rultor/agents/ecs/StartsECS.java

```diff
@@ -79,14 +82,14 @@ public StartsEC2(final Amazon amaz) {
     @Override
     //@todo #629 Add Instance params to Directive, for example publicIpAddress
     public Iterable<Directive> process(final XML xml) throws IOException {
-        final Instance instance = this.amazon.runOnDemand();
+        final ContainerInstance instance = this.amazon.runOnDemand();
         Logger.info(
             this,
-            "EC2 instance %s created",
+            "ECS instance %s created",
             instance
         );
         return new Directives().xpath("/talk")
-            .add("ec2")
-            .attr("id", instance.getInstanceId());
+            .add("ecs")
+            .attr("id", instance.getEc2InstanceId());
```

Comment:
@mkordas ECS is an Amazon EC2 Container Service.

http://docs.aws.amazon.com/AmazonECS/latest/developerguide/ECS_instances.html